component/client-handler/src/test/java/org/wso2/carbon/identity/oauth2/token/handler/clientauth/jwt/util/JWTTestUtil.java

```diff
@@ -214,6 +218,32 @@ public static String signJWTWithRSA(JWTClaimsSet jwtClaimsSet, Key privateKey)
         }
     }
 
+    /**
+     * sign JWT token from PS256 algorithm
+     *
+     * @param jwtClaimsSet contains JWT body
```

Comment:
Complete the java doc comment by explaining the param values and changing the description `sign JWT token from PS256 algorithm` > `Sign JWT token from PS256 algorithm.`

/**
 * Sign JWT token from PS256 algorithm.
 *
 * @param a The first integer.
 * @param b The second integer.
 * @return The sum of the two integers.
 * @throws IllegalArgumentException If either 'a' or 'b' is negative, an IllegalArgumentException is thrown.
 */

Use the above format in all the places where you have java doc comments

Reply:
Fixed in 2c2c165facbf4157db50b88f2765fbc9cee49fe0